app/javascript/packages/badge-tooltip/badge-tooltip-element.ts

```diff
@@ -0,0 +1,71 @@
+import { tooltip } from '@18f/identity-design-system';
+
+class BadgeTooltipElement extends HTMLElement {
+  badge: HTMLElement;
+
+  connectedCallback() {
+    this.badge = this.querySelector('.usa-tooltip')!;
+
+    this.setUpTooltip();
+    this.badge.addEventListener('mouseover', () => this.handleHover());
```

Comment:
In reviewing the default [USWDS Tooltip](https://designsystem.digital.gov/components/tooltip/) behavior, I think we don't actually need to implement all these behaviors, and can instead initialize the default Tooltip. I'm assuming this was mostly copied over from `ClipboardButtonElement`, but the difference with `ClipboardButtonElement` is that we only want the tooltip to be shown after the button is clicked. The default Tooltip will always show when hovering or focusing the element, which is what we want here.

Reply:
> In reviewing the default [USWDS Tooltip](https://designsystem.digital.gov/components/tooltip/) behavior, I think we don't actually need to implement all these behaviors, and can instead initialize the default Tooltip. I'm assuming this was mostly copied over from `ClipboardButtonElement`, but the difference with `ClipboardButtonElement` is that we only want the tooltip to be shown after the button is clicked. The default Tooltip will always show when hovering or focusing the element, which is what we want here.

Converted to use default USWDS Tooltip in 223ff59a96.